### PR TITLE
Removes v1 codebase

### LINKED ISSUE #2047: Get rid of the v1 codebase
We don't have a lot of hands to do "huge maintenance work". There was a lot of demand for a v2 format, so we released a v2 core library as soon as we could (september 2017).

With this in place, we were able to decouple libraries from the v1 codebase. This included brave and zipkin-reporter, which no longer have mandatory dependencies on it. However, there's still a large amount of work to do. This issue tracks that work:

- [x] Port mysql (as-is) to the zipkin2 codebase, or redo it
- [x] Port "cassandra" (as-is) to the zipkin2 codebase, or drop it
- [x] Remove zipkin v1 dep from zipkin-junit and repackage under io.zipkin.zipkin2 namespace
- [x] Make a new collector, which accepts even the old v1 thrift: make the server depend on it.
- [x] redo all the storage tests to port to v2 codebase
- [x] implement the zipkin UI in v2 javascript (implies clock skew and merge logic)

Making this issue was Influenced, but not caused by a strange recent flurry of issues #2044 #2045 #2046 It has always been implicitly on the backlog

### PR DESCRIPTION
**Pull request by codefromthecrypt**
Fixes #2047

### CODE REVIEW

File: zipkin-server/pom.xml [hunk 1]
```diff
@@ -18,11 +18,14 @@
...
 
   <parent>
-    <groupId>io.zipkin.java</groupId>
+    <groupId>io.zipkin.zipkin2</groupId>
     <artifactId>parent</artifactId>
     <version>2.10.2-SNAPSHOT</version>
   </parent>
 
+  <!-- Group ID left alone due to folks downloading it with curl from maven central.
+       TODO: setup a maven redirect -->
```
Comment (devinsba) [inline zipkin-server/pom.xml]: An issue should probably be created for this before merge

File: zipkin-collector/rabbitmq/src/test/java/zipkin2/collector/rabbitmq/ITRabbitMQCollector.java [hunk 2]
```diff
@@ -117,12 +117,12 @@ public void skipsMalformedData() throws Exception {
 
   /** Guards against errors that leak from storage, such as InvalidQueryException */
   @Test
-  public void skipsOnSpanConsumerException() throws Exception {
+  public void skipsOnSpanConsumerException() {
     // TODO: reimplement
   }
 
   @Test
-  public void messagesDistributedAcrossMultipleThreadsSuccessfully() throws Exception {
```
Comment (devinsba) [inline zipkin-collector/rabbitmq/src/test/java/zipkin2/collector/rabbitmq/ITRabbitMQCollector.java:125]: What changed to cause this?

**Review by devinsba — APPROVED** [review]

File: zipkin-collector/rabbitmq/src/test/java/zipkin2/collector/rabbitmq/ITRabbitMQCollector.java (hunk 2 above)
Comment (codefromthecrypt) [reply zipkin-collector/rabbitmq/src/test/java/zipkin2/collector/rabbitmq/ITRabbitMQCollector.java:125]: sorry this was intellij cleanup

File: zipkin-server/pom.xml (hunk 1 above)
Comment (codefromthecrypt) [reply zipkin-server/pom.xml]: https://github.com/openzipkin/zipkin/issues/2132 thanks for the reminder!

**Pull request merged**